### PR TITLE
carina-webdriver: desktop, mobile capabilities unit tests

### PR DESCRIPTION
**Pull request by DenisPitsul2020**

### CODE REVIEW

File: carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/Configuration.java [hunk 1]
```diff
@@ -149,6 +149,8 @@
 
         LOCALE("locale"),
```
Comment (vdelendik) [inline carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/Configuration.java:151]: Language is out-of-dated and shouldn’t be added again!

File: carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/device/Device.java [hunk 2]
```diff
@@ -341,6 +345,7 @@ public void disconnectRemote() {
     public String getFullPackageByName(final String name) {
 
         List<String> packagesList = getInstalledPackages();
```
Comment (vdelendik) [inline carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/device/Device.java:347]: Remove sysout

File: carina-webdriver/src/test/java/com/qaprosoft/carina/core/foundation/webdriver/DevicePoolTest.java [hunk 3]
```diff
@@ -49,4 +49,91 @@ public void registerDevice() {
...
+
+        Assert.assertTrue(device.isNull(), "Device is not null");
+    }
+
+    @Test()
+    public void connectRemoteTest() {
+        Device device = new Device("name", "mobile", "android", "10", "udid", "remoteUrl", "vnc", "proxyPort");
+
+        device.connectRemote();
+        Assert.assertTrue(device.isAdbEnabled(), "Adb commands is disabled");
```
Comment (vdelendik) [inline carina-webdriver/src/test/java/com/qaprosoft/carina/core/foundation/webdriver/DevicePoolTest.java:118]: All isAdbEnabled tests looks strange. How connect/disconnect correlated with this valie?

File: carina-webdriver/src/test/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/impl/desktop/DesktopCapabilitiesTest.java [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+
+import java.util.*;
+
+public class DesktopCapabilitiesTest {
+
+    // default FirefoxProfile preferences
+    private static final boolean MEDIA_EME_ENABLED = true;
+    private static final boolean MEDIA_GMP_MANAGER_UPDATE_ENABLED = true;
+
+    @Test(groups = {"DesktopCapabilitiesTestClass"})
```
Comment (vdelendik) [inline carina-webdriver/src/test/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/impl/desktop/DesktopCapabilitiesTest.java:25]: I don’t see any benefits from declared group

Comment (DenisPitsul2020) [reply carina-webdriver/src/test/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/impl/desktop/DesktopCapabilitiesTest.java:25]: We need to add these groups to specify the order in which the tests will run. If they are not specified in this order, then when building the project locally tests are failed.

**Pull request merged**